replications/internal/queue_management_test.go

```diff
@@ -332,6 +336,66 @@ func TestEnqueueData(t *testing.T) {
 	require.Equal(t, data, string(written))
 }
 
+func TestEnqueueData_WithMetrics(t *testing.T) {
+	t.Parallel()
+
+	queuePath, err := os.MkdirTemp("", "testqueue")
+	require.NoError(t, err)
+	defer os.RemoveAll(queuePath)
+
+	logger := zaptest.NewLogger(t)
+
+	qm := NewDurableQueueManager(logger, queuePath, metrics.NewReplicationsMetrics(), WriteFunc)
+	require.NoError(t, qm.InitializeQueue(id1, maxQueueSizeBytes))
```

Comment:
Nitpick, but is there a reason you didn't use the test helper `initQueueManager()`?

Reply:
Nope! Thanks for the suggestion; that should simplify the test, I'll update it to use `initQueueManager()`.